python/cudf/cudf/_typing.py

```diff
@@ -32,3 +32,9 @@
 SeriesOrSingleColumnIndex = Union[
     "cudf.Series", "cudf.core.index.GenericIndex"
 ]
+
+# Groupby aggregation
+AggType = Union[str, Callable]
+MultiColumnAggType = Union[
+    AggType, Iterable[AggType], Dict[Any, Iterable[AggType]]
+]
```

Comment:
I'm fine with these for now. Eventually we're probably going to want to revamp a lot of our typing, but no harm in making incremental improvements where we can.

Reply:
I fought pretty hard with mypy to get these to work :rofl: In the documentation Pandas says a collection of aggregations are to be specified with a list, but in reality any iterable would do.

Reply:
I'm glad this works now then :)